sqlx-macros/src/database/postgres.rs

```diff
@@ -88,9 +88,8 @@ impl_database_ext! {
         #[cfg(feature = "chrono")]
         Vec<sqlx::types::chrono::NaiveDateTime> | &[sqlx::types::chrono::NaiveDateTime],
 
-        // TODO
-        // #[cfg(feature = "chrono")]
-        // Vec<sqlx::types::chrono::DateTime<sqlx::types::chrono::Utc>> | &[sqlx::types::chrono::DateTime<_>],
+        #[cfg(feature = "chrono")]
+        Vec<sqlx::types::chrono::DateTime<sqlx::types::chrono::Utc>> | &[sqlx::types::chrono::DateTime<_>],
```

Comment:
This isn't enough. You also need to add the following trait implementations which don't exist yet:

* `Decode, Encode, Type` for `Vec<DateTime<Utc>`
* `Encode, Type` for `&[DateTime<Tz>]` where `Tz: TimeZone`